src/lobby/react.tsx

```diff
@@ -42,16 +86,7 @@ const LobbyPhases = {
  *   A React component that provides a UI to create, list, join, leave, play or
  *   spectate matches (game instances).
  */
-class Lobby extends React.Component {
-  static propTypes = {
```

Comment:
Could we keep prop types for this public-facing component? Keeping prop types gives some development warnings for developers who aren’t using Typescript, which might be better than nothing for them.

Reply:
Done in 3bacaef.